DOCS.org

```diff
@@ -59,17 +59,56 @@
 
 ** directory
 
-   Path to the directory containing org files for use with org-roam.
+   Path to the directory containing org files for use with org-roam and where
+   org-roam stores org files which represent new nodes created with org-roam.
 
    Takes a string representing the path to the directory, which can
    can contain =~= that will expand to your home directory.
 
    #+begin_src lua
    require("org-roam").setup({
-     directory = "~/orgfiles",
+     directory = "~/orgroamfiles",
    })
    #+end_src
 
+   Configuring additional folders and files to be loaded by org-roam is also
+   supported, see [[#configuration-org_files][org_files]]:
+
+** org_files
+
+   To include existing org files in the knowledge graph of org roam, additional
+   files can be registered. Supported syntax is like =org_agenda_files= in
+   orgmode. If the entry does not end on =.org= a directory is assumed and
+   searched recursively for org files.
+
+   #+begin_src lua
+   require("org-roam").setup({
+     org_files = {
+       "~/more_org_files",
+       "~/additional/org_files/*.org",
+     },
+   })
+   #+end_src
+
+   To actually connect the content of these files with org-roams knowledge
+   graph, you can use the "store link" feature of orgmode (=,ols=), which adds
```

Comment:
You have the keybinding as `,ols` as I assume the default leader key is comma. Can we show it as `<Leader>ols` instead?

Reply:
You are absolutely correct. I confused myself here, actually I have the prefix of orgmode mapped to just `,` and the default prefix is `<Leader>o`. `,o` is completely wrong, it is a mash-up of my mapping and the default one. :sweat_smile: